tests/specs/benchmarks/overflow00-nooverflow-spec.k

```diff
@@ -110,23 +110,21 @@ _
       </network>
     </ethereum>
     requires
-     #rangeAddress(#CONTRACT_ID)
-     andBool #rangeAddress(#CALLEE_ID)
+     #rangeAddress(CONTRACT_ID)
+     andBool #rangeAddress(CALLEE_ID)
      andBool #rangeUInt(256, NOW)
      andBool #rangeUInt(128, BLOCK_NUM) // Solidity
 
      // Account address normality
-     andBool #CONTRACT_ID  >Int 0 andBool (notBool #CONTRACT_ID  in #precompiledAccounts(BYZANTIUM))
-     andBool #CALLEE_ID  >Int 0 andBool (notBool #CALLEE_ID  in #precompiledAccounts(BYZANTIUM))
+     andBool CONTRACT_ID  >Int 0 andBool (notBool CONTRACT_ID  in #precompiledAccounts(BYZANTIUM))
+     andBool CALLEE_ID  >Int 0 andBool (notBool CALLEE_ID  in #precompiledAccounts(BYZANTIUM))
 
 andBool #rangeUInt(256, CONTRACT_BAL)
-andBool #rangeUInt(256, CALLEE_BAL)andBool #range(0 <= CD < 1024)
-andBool #rangeAddress(MSG_SENDER)andBool #range(0 <= A0 < 2 ^Int 256 -Int 1)
-andBool N ==Int A0 +Int 1
+andBool #rangeUInt(256, CALLEE_BAL) andBool #range(0 <= CD < 1024)
+andBool #rangeAddress(MSG_SENDER) andBool #range(0 <= A0 < 2 ^Int 256 -Int 1)
     ensures true
```

Comment:
@daejunpark is there any reason to keep `ensures true`? Or should we remove that too?

Reply:
I think this was auto-generated. But having this makes it easier to move some constraints from `requires` to `ensures` and back, because we don't need to remove `andBool` and add back.  (Indeed, that happened for the haskell backend porting, because the haskell backend strictly distinguishes `requires` from `ensures`, while the java backend does not.) So, I think it is not that bad to keep this dummy. But I'm OK with removing this, if you want.